test/CMakeLists.txt

```diff
@@ -2297,7 +2297,7 @@ ecbuild_add_test( TARGET  test_${PROJECT_NAME}_smap9km_ssm
                     COMMAND bash
                     ARGS    ${CMAKE_BINARY_DIR}/bin/iodaconv_comp.sh
                             netcdf
-                            "${Python3_EXECUTABLE} ${CMAKE_BINARY_DIR}/bin/smap9km_ssm2ioda.py
+                            "${Python3_EXECUTABLE} ${CMAKE_BINARY_DIR}/bin/smap_ssm2ioda.py
                             -i testinput/SMAP_L2_SM_P_E_36365_D_20211122T013945_R18240_001.h5
                             -o testrun/smap9km_ssm.nc
```

Comment:
this is also our chance to change these names... I still don't follow what the difference between the `9km` test which does NOT use the near real time `NRT` file

should we rename the `smap9km` to something else?  

or is it the other way around should we name these files and this test just `smap_ssm` and the other test using the `NRT` file the smap_NRT_ssm ?

I can do that if you like

Reply:
in a new issue and PR will take on removal of defunct routine `smap9km_ssm2ioda.py` and the change names of test files to be either plan `smap_ssm` or `smap_NRT_ssm` to distinguish